profiling/tests/wordpress.pprof

```diff
@@ -0,0 +1,4441 @@
+
```

Comment:
Consider perhaps compressing this file as `pprof.gz` or something OR at least telling github not to render them https://thoughtbot.com/blog/github-diff-supression :)

Reply:
Good idea 👍🏻